CHANGES/801.bugfix

```diff
@@ -0,0 +1 @@
+Fix pulpcore deploy dir
```

Comment:
Also, please phrase this better.

Something like "Fixed pulp_installer failing when variables like pulp_user_home are set by specifying them in /etc/pulp/settings.py"